tasks/create_rhaptos_site.yml

```diff
@@ -42,3 +42,28 @@
 - name: start instance
   become: yes
   shell: supervisorctl restart plone-instance
+
+- name: send the script to create list of zope urls
+  when: rhaptos_site|failed
+  become: yes
+  template:
+    src: "misc/zope_urls_from_db.py"
```

Comment:
Any reason we shouldn't put this script in the cnx-buildout repo? Seems to me to be more aligned with that repo's purpose and existence.